cms/server/admin/handlers/contestuser.py

```diff
@@ -140,9 +140,12 @@ def get(self, contest_id, user_id):
         if participation is None:
             raise tornado.web.HTTPError(404)
 
-        self.r_params = self.render_params()
+        submission_query = self.sql_session.query(Submission)\
+            .filter(Submission.participation_id == participation.id)
```

Comment:
Any reason to put the equality on the IDs instead of on the objects themselves? I think we usually prefer the latter if there's no big disadvantage... (e.g. an additional query firing)